tests/openfl/utilities/test_verifiable_dataset.py

```diff
@@ -0,0 +1,263 @@
+import json
+from openfl.utilities.verifdataset.local_data_source import LocalDataSource
+from openfl.utilities.verifdataset.verifiable_dataset import DatasetFormat, VerifiableDataset
+import pytest
+import os
+import shutil
+from pathlib import Path
+from typing import List, Tuple
+
+@pytest.fixture(scope="class")
+def data_sources(tmp_path_factory) -> Tuple[Path, Path]:
```

Comment:
can you add a directory with a symlink to test the realpath ?

Reply:
see above

Reply:
I mean as a unit test

Reply:
currently there are the test cases:
test_two_local_datasource_concise_with_symlink
test_two_local_datasource_different_base_dir_concise_with_symlink
test_two_local_datasource_verbose_with_symlink
test_two_local_datasource_different_base_dir_verbose_with_symlink
 (in test__verifiable_dataset.py)

what do I miss?